src/shell-integration/fish/vendor_conf.d/ghostty-shell-integration.fish

```diff
@@ -49,10 +49,10 @@ status --is-interactive || ghostty_exit
 function __ghostty_setup --on-event fish_prompt -d "Setup ghostty integration"
     functions -e __ghostty_setup
 
-    # Check if we are setting cursors
-    set --local no_cursor "$GHOSTTY_SHELL_INTEGRATION_NO_CURSOR"
+    set --local cursor string match -q "*cursor*" "$GHOSTTY_SHELL_FEATURES"
+    set --local sudo string match -q "*sudo*" "$GHOSTTY_SHELL_FEATURES"
```

Comment:
For Fish at least I don't think this is the right approach. Fish has builtin list types and string operation primitives that guarantee the env var must be in a comma-delimited string. A combination of `string split` and `contains` makes much more sense IMO.

Reply:
That's a good idea. Thanks for the suggestion!